fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/model/IDatabaseObject.java

```diff
@@ -108,11 +108,11 @@
     public String getName();
 
     /**
-     * Get the qualified name for this object prefixed with the object type
-     * which acts as a namespace
+     * Get the qualified name for this object:
+     *   objectType:objectName:objectVersion
      * @return
      */
-    public String getTypeAndName();
+    public String getTypeNameVersion();
```

Comment:
does this signature change impact anything in prior releases?
e.g. what's stored in the version history table?

Reply:
No, this doesn't change the semantics of the version history table. Its primary use is to provide a unique name describing the object/change in the task list (which can apply schema changes in parallel).